src/utils/gaa.js

```diff
@@ -984,16 +984,19 @@ export class GaaSignInWithGoogleButton {
       });
     })
       .then((jwt) => {
-        const jwtPayload = /** @type {!GoogleIdentityV1} */ (
-          new JwtHelper().decode(jwt.credential)
-        );
+        let returnedJwt = jwt;
+        if (!rawJwt) {
+          returnedJwt = /** @type {!GoogleIdentityV1} */ (
+            new JwtHelper().decode(jwt.credential)
+          );
+        }
 
         // Send GAA user to parent frame.
         sendMessageToParentFnPromise.then((sendMessageToParent) => {
           sendMessageToParent({
             stamp: POST_MESSAGE_STAMP,
             command: POST_MESSAGE_COMMAND_USER,
-            jwtPayload,
+            returnedJwt,
```

Comment:
since `jwtPayload` is in deployed code we should continue to return it and mark it deprecated in a comment so we don't break anything.

Reply:
I've updated the code to note that `jwtPayload` is being deprecated.